docs/README.ar.md

```diff
@@ -0,0 +1,68 @@
+<h3 align="center"><img src="../assets/onefetch.svg" height="130px"></h3>
+
+<h5 align="center">Rust مكتوبه بالغة GIT أداة سطر الأوامر لعرض معلومات مستودع</h5>
+
+<p align="center">
+	<a href="https://crates.io/crates/onefetch"><img src="https://img.shields.io/crates/v/onefetch.svg" alt="cargo"></a>
+	<a href="https://github.com/o2sh/onefetch/actions"><img src="https://github.com/o2sh/onefetch/workflows/CI/badge.svg" alt="Build Status"></a>
+  <a href="https://onefetch.dev"><img src="../assets/language-badge.svg"></a>
+	<a href="https://github.com/o2sh/onefetch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22"><img src="https://img.shields.io/github/issues/o2sh/onefetch/help%20wanted?color=green" alt="help wanted"></a>
+	<a href="../LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+	<img src="../assets/msrv-badge.svg">
+</p>
+
+<img src="../assets/screenshot-1.png" align="left" height="240px">
+
+<p dir="rtl" lang="ar">
+Onefetch
+هي أداة سطر الأوامر مكتوبة بالغة 
+Rust
+تعرض معلومات المشروع وإحصائيات الكود لمستودع
+Git
+.المحلي مباشرة الي جهازك الطرفي    
+الأداة مستقلة تمامًا عن الإنترنت - لا يتطلب الوصول إلى الشبكة.
+</p>
+
+<p dir="rtl" lang="ar">
+يتم عرض معلومات المستودع جنباً إلي جنب مع شعار اللغة السائدة في المشروع بشكل إفتراضي، ولكن يمكنك أيضاً تكوين
+Onefetch
+بحيث يستخدم بدلاً من ذلك صورة - على الأنظمة المدعومة - أو إدخال نص أو حتى لا شيء على الإطلاق.
+</p>
+
+<p dir="rtl" lang="ar">
+يكتشف تلقائياً التراخيص مفتوحة المصدر من النصوص، ويزود المستخدم بمعلومات قيمة مثل توزيع الكود، التغييرات المعلقة، عدد التبعيات (بواسطة مدير الحزم)، المساهمين الأساسين (حسب عدد الالتزامات)، وحجم المستودع علي القرص الصلب، وتاريخ الإنشاء،
+LOC
+(عدد أسطر الكود)، إلخ.
+</p>
+
+<img src="../assets/screenshot-2.png" align="left" height="240px">
+
+<p dir="rtl" lang="ar">
+يمكن تكوين
+Onefetch
+عن طريق معلمات سطر الأوامر
+(Flags)
+لعرض ما تريده بالضبط، بالطريقة التي تريدها.
+يمكنك تخصيص تنسيق النص/
+ASCII
+، وتعطيل المعلومات التي لا تهمك، وتجاهل الملفات والمجلدات، وإخراج النتائج بتنسيقات مختلفة
+مثل
+(Json, Yaml)
+إلخ.
+</p>
+
+<p dir="rtl" lang="ar">
+حتى الآن، يدعم
+Onefetch
+أكثر من
+<a href="https://onefetch.dev">100 لغة برمجة مختلفة</a>
```

Comment:
minor: I was waiting for the 101th language to update all the READMEs accordingly - to be correct with the "more than" part.

Reply:
Ok, I'll be return it to 50 – to be easiest to replace when reach to the 101st language